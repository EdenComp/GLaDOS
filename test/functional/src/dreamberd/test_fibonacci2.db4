fn fibonacci(n) {
    if (n == 0) {
        return 0;
    } elif (n == 1) {
        return 1;
    } else {
        int a1 = fibonacci(n - 1);
        int a2 = fibonacci(n - 2);
        int a3 = a1 + a2;
        return (a3);
    }
}

fn main() {
    var number = 12;

    print("Série de Fibonacci jusqu'à " + number + " termes:\n");
    for (var i = 0; i < number; i = i + 1) {
        print(fibonacci(i));
        if (i < number - 1) {
            print(", ");
        }
    }
}

main();
