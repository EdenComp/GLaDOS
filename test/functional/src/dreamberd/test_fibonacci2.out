Série de Fibonacci jusqu'à 12 termes:
0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89